Moodis/Moodis/Feature/Login/LoginViewModel.cs

```diff
@@ -10,10 +10,12 @@ namespace Moodis.Feature.Login
 {
     class LoginViewModel
     {
+        public static List<User> userList;
+        public static User currentUser;
```

Comment:
Why these properties are static?

Reply:
there should only be 1 instance of current user and 1 instance of userlist. These should later down the line be put in a class where we can store current users and their lists.